components/src/core/components/Input/Select/SelectInputButton.vue

```diff
@@ -2,6 +2,7 @@
   <div
     class="oxd-select-wrapper"
     :class="{'d-flex justify-center': hideDropdownLabel}"
+    v-click-outside="clickOutside"
```

Comment:
Added the missing directive "click-outside". This got missing when I was changing the UI of the vacancy dropdown